bitnami/dataplatform-bp2/values.yaml

```diff
@@ -16,6 +16,536 @@ global:
+      type: ""
+      ## @param dataplatform.exporter.nodeAffinityPreset.key Data Platform exporter node label key to match Ignored if `affinity` is set.
+      ## E.g.
+      ## key: "kubernetes.io/e2e-az-name"
+      ##
+      key: ""
+      ## @param dataplatform.exporter.nodeAffinityPreset.values Data Platform exporter node label values to match. Ignored if `affinity` is set.
+      ## E.g.
+      ## values:
+      ##   - e2e-az1
+      ##   - e2e-az2
+      ##
+      values: []
+    ## @param dataplatform.exporter.affinity Affinity settings for exporter pod assignment. Evaluated as a template
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ##
+    affinity: {}
+    ## @param dataplatform.exporter.nodeSelector Node labels for exporter pods assignment. Evaluated as a template
+    ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+    ##
+    nodeSelector: {}
+    ## @param dataplatform.exporter.tolerations Tolerations for exporter pods assignment. Evaluated as a template
+    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    ##
+    tolerations: []
+    ## @param dataplatform.exporter.podLabels Additional labels for Metrics exporter pod
+    ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+    ##
+    podLabels: {}
+    ## @param dataplatform.exporter.podAnnotations Additional annotations for Metrics exporter pod
+    ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+    ##
+    podAnnotations: {}
+    ## @param dataplatform.exporter.customLivenessProbe Override default liveness probe%%MAIN_CONTAINER_NAME%%
+    ##
+    customLivenessProbe: {}
+    ## @param dataplatform.exporter.customReadinessProbe Override default readiness probe%%MAIN_CONTAINER_NAME%%
+    ##
+    customReadinessProbe: {}
+    ## Update strategy
+    ## If replicas = 1, an update can get "stuck", as the previous pod remains attached to the
+    ## PV, and the "incoming" pod can never start. Changing the strategy to "Recreate" will
+    ## terminate the single previous pod, so that the new, incoming pod can attach to the PV
+    ## @param dataplatform.exporter.updateStrategy.type Update strategy - only really applicable for deployments with RWO PVs attached
+    ## @param dataplatform.exporter.updateStrategy.rollingUpdate Deployment rolling update configuration parameters
+    ##
+    updateStrategy:
+      type: RollingUpdate
+      rollingUpdate: {}
+    ## @param dataplatform.exporter.extraEnvVars Additional environment variables to set
+    ## Example:
+    ## extraEnvVars:
+    ##   - name: FOO
+    ##     value: "bar"
+    ##
+    extraEnvVars: []
+    ## @param dataplatform.exporter.extraEnvVarsCM ConfigMap with extra environment variables
+    ##
+    extraEnvVarsCM: ""
```

Comment:
Add this to the deployment, you can see an example in the example yaml in the template/CHART_NAME/templates/deployment.yaml